packages/core/helper-plugin/lib/src/components/PaginationURLQuery/tests/__snapshots__/index.test.js.snap

```diff
@@ -554,7 +554,7 @@ exports[`<PaginationURLQuery /> should work when the pageCount is inferior or eq
         aria-current="page"
         aria-disabled="true"
         class="c2 c3 active"
-        href="/test"
+        href="/test?page=0&pageSize=10&sort=firstname"
```

Comment:
Same here 🤔

Reply:
I guess it's not a big issue as the link is disabled but so the user will never be able to go to page 0. What's weird is that it did not use to work this way previously. It might be because of the new implementation in the DS.